pkg/fakerp/server.go

```diff
@@ -208,28 +208,31 @@ func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, resp, http.StatusInternalServerError)
 		return
 	}
-	go func() {
-		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
-		defer cancel()
-		if err := future.WaitForCompletionRef(ctx, gc.Client); err != nil {
-			resp := "500 Internal Error: Failed to wait for resource group deletion"
-			s.log.Debugf("%s: %v", resp, err)
-			return
-		}
-		resp, err := future.Result(gc)
-		if err != nil {
-			resp := "500 Internal Error: Failed to get resource group deletion response"
-			s.log.Debugf("%s: %v", resp, err)
-			return
-		}
-		// If the resource group deletion is successful, cleanup the object
-		// from the memory so the next GET from the client waiting for this
-		// long-running operation can exit successfully.
-		if resp.StatusCode == http.StatusOK {
-			s.log.Infof("deleted resource group %s", resourceGroup)
-			s.write(nil)
-		}
-	}()
+	// no-wait for CI
+	if os.Getenv("NO_WAIT") == "" {
```

Comment:
I don't think we need to do this server-side. Clients choose whether they want to wait or not.

Reply:
removed. missed client side check.